### PR TITLE
Prepare release v0.12.0

### PR DESCRIPTION
**Pull request by uint**

### CODE REVIEW

**Review by ethanfrey — APPROVED** [review]
Looks good to me, but I would wait for @maurolacy to give a 👍

**Review by ueco-jb — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,19 +2,44 @@
...
+- Allow modifying admin of Ics20 contract [\#641](https://github.com/CosmWasm/cw-plus/pull/641) ([ethanfrey](https://github.com/ethanfrey))
+- `MIGRATING.md` update / examples for type safe bounds [\#640](https://github.com/CosmWasm/cw-plus/pull/640) ([maurolacy](https://github.com/maurolacy))
 - Fix benchmarks \(after 1.58.1 update\) [\#639](https://github.com/CosmWasm/cw-plus/pull/639) ([maurolacy](https://github.com/maurolacy))
 - Fix `remove_hook` helper [\#638](https://github.com/CosmWasm/cw-plus/pull/638) ([maurolacy](https://github.com/maurolacy))
 - Type safe bounds [\#627](https://github.com/CosmWasm/cw-plus/pull/627) ([maurolacy](https://github.com/maurolacy))
 
+## [juno-ics20](https://github.com/CosmWasm/cw-plus/tree/juno-ics20) (2022-01-27)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.12.0-alpha1...juno-ics20)
+
```
Comment (maurolacy) [inline CHANGELOG.md:42]: This can be removed.

**Review by maurolacy — APPROVED** [review]
LGTM.

File: CHANGELOG.md (hunk 1 above)
Comment (uint) [reply CHANGELOG.md]: Good catch, done

**Pull request merged**